fmn/consumer/digests.py

```diff
@@ -0,0 +1,134 @@
+# FMN worker calculating and sending digests
+
+
+import json
+import logging
+import time
+import random
+
+import pika
+import fedmsg
+import fedmsg.meta
+import fedmsg_meta_fedora_infrastructure
+
+from pika.adapters import twisted_connection
+from twisted.internet import defer, reactor, protocol,task
+
+import fmn.lib
+import fmn.rules.utils
+import fmn.consumer.backends as fmn_backends
+import fmn.consumer.producer as fmn_producers
+
+import fmn.consumer.fmn_fasshim
+from fedmsg_meta_fedora_infrastructure import fasshim
+
+log = logging.getLogger("fmn")
+log.setLevel('DEBUG')
+CONFIG = fedmsg.config.load_config()
+fedmsg.meta.make_processors(**CONFIG)
+
+DB_URI = CONFIG.get('fmn.sqlalchemy.uri', None)
+session = fmn.lib.models.init(DB_URI)
+
+fmn.consumer.fmn_fasshim.make_fas_cache(**CONFIG)
+# Monkey patch fedmsg_meta modules
```

Comment:
That's to allow  fedmsg_meta use FMN's cache instead of querying FAS every time it needs info from it.